contributors/design-proposals/controller_history.md

```diff
@@ -0,0 +1,447 @@
+# Controller History
+
+**Author**: kow3ns@
+
+**Status**: Proposal
+
+## Abstract 
+In Kubernetes, in order to update and rollback the configuration and binary 
+images of controller managed Pods, users mutate DaemonSet, StatefulSet, 
+and ReplicaSet (via Deployment) Objects, and the corresponding controllers 
```

Comment:
Yes. This is not clear. You mutate Deployments which causes replicaset's to be created, deleted, and updated. It is a more complicated workflow. I will update to reflect this.